src/taipy/core/config/__init__.py

```diff
@@ -71,9 +74,16 @@
         ("configure_scenario_from_tasks", ScenarioConfig._configure_from_tasks),
     ],
 )
+_inject_section(
+    VersionMigrationConfig,
+    "version_migrations",
```

Comment:
I like `MigrationConfig`.

But `migration_functions` is not sufficient to describe the config here.
Because when we have multiple "`migration_functions`" (or multiple `MigrationConfig`), each migration function actually contains multiple functions.

Reply:
Yes sorry, I though it was a UniqueSection, and not a Section.